packages/palette/src/elements/Modal/Modal.tsx

```diff
@@ -11,6 +11,8 @@ import { Serif } from "../Typography"
 
 interface ModalProps {
   FixedButton?: JSX.Element
+  // contentAnimationKey should change if the modal displays new content
+  contentAnimationKey?: string
```

Comment:
This is a bit opaque naming wise -- maybe `refresh` or something a little more user friendly?